.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,33 @@
+---
+name: RFE Proposal
```

Comment:
what's RFE?

Reply:
Idk... it was in the CAPI template

Reply:
can we rename it to something that's more meaningful to us? Maybe `Enhancement Proposal`?

Reply:
Request for enhancement

Reply:
RFE, spell out RFE, or `Enhancement Proposal`? Ready... vote!

Reply:
I vote `Enhancement Proposal`, I don't think acronyms are friendly to newcomers. And if I we both didn't know what it was, chances are some contributors won't know either.

Reply:
Anyone else have a preference?

Reply:
`Enhancement Proposal`

Reply:
I've changed the issue template to be titled `Enhancement Proposal`.